.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
+    "extends": "airbnb",
+
+    "rules": {
+        "indent": [2, 2],
```

Comment:
Good question! Our motivation with this PR is to reduce barriers to other front-end devs from the open source community contributing to Wagtail. One reasonably consistent feature of the JS ecosystem is two space indentation. We felt that it would be more appropriate to implement a broader community standard (airbnb) rather than our personal preference (I'm also not fond of two spaces).